src/components/Editor.jsx

```diff
@@ -78,14 +83,19 @@ class Editor extends React.Component {
       // would evaluate to false
       this.emphasizeLine(openLine);
     }
-    if (value && AST.children && filters) {
+    if (newAST || newFilters) {
```

Comment:
You need to flip these after the highlight occurs, or else it will keep highlighting it.